<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSettings</class>
 <widget class="QWidget" name="WidgetSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icons/ressources/settings.png</normaloff>:/icons/ressources/settings.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="layMain">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabBasicSettings">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="layBasic" stretch="0,0,0,0,2">
       <item>
        <widget class="QLabel" name="lblNick">
         <property name="text">
          <string>Nick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txtNick">
         <property name="maxLength">
          <number>256</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lblShared">
         <property name="text">
          <string>Shared folders</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0">
         <item>
          <widget class="QPushButton" name="butAddShared">
           <property name="toolTip">
            <string>Add a new shared directory</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/icons/ressources/add.png</normaloff>:/icons/ressources/add.png</iconset>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butRemoveShared">
           <property name="toolTip">
            <string>Remove the selected shared directory</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/icons/ressources/delete.png</normaloff>:/icons/ressources/delete.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butMoveDownShared">
           <property name="toolTip">
            <string>Move down the selected shared directory</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/icons/ressources/arrow_down.png</normaloff>:/icons/ressources/arrow_down.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butMoveUpShared">
           <property name="toolTip">
            <string>Move up the selected shared directory</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/icons/ressources/arrow_up.png</normaloff>:/icons/ressources/arrow_up.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="butOpenFolder">
           <property name="toolTip">
            <string>Open location of the selected shared directory</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/icons/ressources/explore_folder.png</normaloff>:/icons/ressources/explore_folder.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layShared">
         <item>
          <widget class="QTableView" name="tblShareDirs">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAdvancedSettings">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="layAdvanced" stretch="0,0,0,0,0">
       <item>
        <widget class="QLabel" name="lblLanguage">
         <property name="text">
          <string>Language</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbLanguages"/>
       </item>
       <item>
        <widget class="QGroupBox" name="grpCoreConnection">
         <property name="title">
          <string>Core connection</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lblCoreAddress">
            <property name="text">
             <string>address</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="txtCoreAddress"/>
            </item>
            <item>
             <widget class="QPushButton" name="butResetCoreAddress">
              <property name="text">
               <string>Reset to local</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="lblPassword">
            <property name="text">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPassword">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkEnableIntegrityCheck">
         <property name="toolTip">
          <string>When activated all received data will be verified against their hashes.  It may consume CPU resources.</string>
         </property>
         <property name="text">
          <string>Data integrity check</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
